docs/proposals/README.md

```diff
@@ -27,12 +30,34 @@ GEPs should be created as Markdown `.md` files and are submitted through a GitHu
 
 If not already done, please present your GEP in the [regular community meeting](https://github.com/gardener/documentation/blob/master/CONTRIBUTING.md#weekly-meeting) to brief the community about your proposal (we strive for personal communication :) ). Also consider that this may be an important step to raise awareness and understanding for everyone involved.
 
+The GEP template contains a small set of metadata, which is helpful for keeping track of the enhancement
+in general and especially of who is responsible for implementing and reviewing PRs that are part of
+the enhancement.
+
+### Main Reviewers
+
+Apart from general metadata the GEP should name at least one "main reviewer".
```

Comment:
Can you elaborate in the text how community members are supposed to find a main reviewer?

Reply:
Makes sense. Added a sentence, WDYT?

Reply:
Nice, thank you :)